packages/integration/lib/likeminds_feed_core.dart

```diff
@@ -54,6 +54,10 @@ class LMFeedCore {
 
   static LMFeedWidgetUtility get widgetUtility => instance._widgetUtility;
 
+  static set widgetUtility(LMFeedWidgetUtility value) {
```

Comment:
Shouldn't we have similar setter and getter for theme and config as well? We are calling initialise again on Cred screen to change the theme

Reply:
we can make getter and setter for both of theme,_widgetUtlilty was marked private so it was necessary to make a getter and setter to access these value outside the class but theme and config are marked as public variable and also there is a function initialiseTheme(), which acts similer to getter for theme.